lib/pluginmanager/install.rb

```diff
@@ -43,9 +43,11 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    Gem.sources = Gem::SourceList.from(gemfile.gemset.sources)
     gems.each do |plugin, version|
       puts("Validating #{[plugin, version].compact.join("-")}")
-      signal_error("Installation aborted, verification failed for #{plugin} #{version}") unless LogStash::PluginManager.logstash_plugin?(plugin, version)
+      next unless LogStash::PluginManager.logstash_plugin?(plugin, version)
+      signal_error("Installation aborted, verification failed for #{plugin} #{version}")
```

Comment:
I am not sure the logic is correct here. Previously it was:
"signal error if not a valid logstash plugin"

and the new logic is now:
"proceed to next check if not a valid logstash plugin otherwise continue and signal error (if it is a valid plugin)"

was this tested? also, the proposed line breakup of using `next unless ...` is a lot harder to understand and follow the logic.